### PR TITLE
Lines reducing in code

### PR DESCRIPTION
**Pull request by PraveenNanda124**
Reducing some lines of code in some files

### CODE REVIEW

File: src/main/java/com/thealgorithms/backtracking/KnightsTour.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
     Input : N = 8
 
     Output:
-        0  59  38  33  30  17   8  63
+        0   59  38  33  30  17   8  63
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/backtracking/KnightsTour.java:16]:
```suggestion
        0  59  38  33  30  17  8  63
```
Remove the extra space

File: src/main/java/com/thealgorithms/backtracking/KnightsTour.java [hunk 2]
```diff
@@ -105,8 +98,7 @@ private static List<int[]> neighbors(int row, int column) {
         List<int[]> neighbour = new ArrayList<>();
 
         for (int[] m : moves) {
-            int x = m[0];
-            int y = m[1];
+            int x = m[0],y = m[1];
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/backtracking/KnightsTour.java:101]: Remove this change.

https://www.oracle.com/java/technologies/javase/codeconventions-declarations.html

**Review by debasishbsws — COMMENTED** [review]
https://stackoverflow.com/questions/24729206/whats-java-standard-for-single-line-if-statement

Should check this.

**Comment by siriak** [comment]
The proposed changes do not follow the code style of this repository. Having braces is fine!

**Pull request closed without merging**